README.md

```diff
@@ -94,28 +94,23 @@ where you replace UDID and PackageID with their values you found before.
 
 
 
-1. Download [https://GitHub.com/jkcoxson/DiOS-instructions](https://GitHub.com/jkcoxson/DiOS-instructions)
+1. **Make sure you have a github account logged in beforehand** and Download the latest one that says *master* on the right and make sure it is MacOS and Extract it in Downloads folder. 
 
-2. Move the two idevice files to 
+	[Download](https://github.com/libimobiledevice/libimobiledevice/actions)
 
-`` /usr/local/bin ``
+2. In terminal enter 
 
-3. Move the dylib to 
-`` /usr/local/lib ``
+`` chmod +x ~/Downloads/libimobiledevice-latest_macOS/libimobiledevice.tar ``
```

Comment:
I was saying that the bins have to be chmod-ed, not the tar

Reply:
> I was saying that the bins have to be chmod-ed, not the tar

will resolve